app/react/Templates/components/specs/MetadataTemplate.spec.js

```diff
@@ -173,14 +177,34 @@ describe('MetadataTemplate', () => {
   });
 
   describe('onSubmit', () => {
-    it('should thrim the properties labels and then call props.saveTemplate', () => {
+    it('should thrim the properties labels and then call props.saveTemplate', async () => {
```

Comment:
`Thrim` is a great word, unfortunately not on the English language :).  I know it was there already, but since you are editing these files, please fix this as well? Thanks.